packages/cli/src/lib/bundler/config.ts

```diff
@@ -177,6 +178,7 @@ export async function createConfig(
       extensions: ['.ts', '.tsx', '.mjs', '.js', '.jsx'],
       mainFields: ['browser', 'module', 'main'],
       fallback: {
+        ...pickBy(require('node-libs-browser')),
```

Comment:
This is not very clear :)

Reply:
BUT THE PERF!1

I blame lodash for providing that silly default, can't bear to provide an arg that is identical to the default x)

Reply:
sure but, `const cloneWithoutFalsyFields = x` or whatnot :)

Reply:
just didn't understand the underlying purpose of it at first glance